app/app.go

```diff
@@ -84,7 +85,7 @@ import (
 	dbm "github.com/tendermint/tm-db"
 )
 
-const Name = "ChainLink"
+const Name = "chainlink"
```

Comment:
does the app name need to be app lower case?

Reply:
yes, it should be all lower case, if there is an err with the cli cmd, it'll give the wrong cmd name

Reply:
good to know, tHanKs @araskachoi 😆